docs/client-authorization.md

```diff
@@ -0,0 +1,24 @@
+---
+id: client-authorization
+title: NKN Client Authorization
+---
+
+When a NKN client joins NKN network, it sends a RPC request to seed list to enquiry which node that it should connect by webscoket.
+
+After get the webscoket node infomation, the client connects to the node, and send its client Id, public key to the node.
+
+Then the node need check that the client is the right client who owns this client Id and public. This is client authorization for.

```

Comment:
need -> needs to
This is client authorization for -> This is what client authorization is for